e2e/game_page.js

```diff
@@ -0,0 +1,15 @@
+"use strict";
+import { Selector } from 'testcafe'; // first import testcafe selectors
+
+export default class GamePage {
+    constructor () {
+        this.hand = Selector('.hand');
+        this.cardsInHand = this.hand.find('.card');
+        this.joinLink = Selector('#join-link');
+        this.rows = Selector('.game-row');
+        this.myScore = Selector('#my-score');
```

Comment:
`myScore` and `opponentScore` is how many rows are won?
Given that we have 6 scores (points on each row and number of rows won) we should try to be more clear regarding that.

Reply:
Currently they don't work (there's no test that actually tests them) but that would be the idea.